image-filters/image-filters.markdown

```diff
@@ -573,56 +570,56 @@ of colors in a program is very important to understanding how our filters work.
 To prepare ourselves for working on our next filter, let's explore the concept
 of color a bit more.
 
-You may not have known it at the time, but we were unwittingly using a concept
-in the previous section called a "color space", which is way of representing
-color digitally. Kids mixing paints learn that all colors can be made from
-other colors, but things work slightly differently in digital (less risk of
+We were using a concept
+in the previous section called "color space", which is way of representing
+color digitally. Kids mixing paints learn that colors can be made from
+other colors; things work slightly differently in digital (less risk of
 being covered in paint!) but similar. Processing makes it really easy to work
 with whatever color space you want, but you need to know which one to pick, so
 it’s important to understand how they work.
 
 #### RGB colors
-The color space that most programmers are familiar with is RGBA - red, green,
-blue and alpha. In hexadecimal (base 16) the first two digits are the amount of
-red the second two blue, the third two green, and the final two (if they are
-there) are the alpha value. These values range from 00 in base 16, 0 in base
-10, through to FF, the equivalent of 255 in base 10. The alpha represents the
-opacity, where 0 is transparent, and 100% opaque.
+The color space that most programmers are familiar with is RGBA: red, green,
+blue and alpha. In hexadecimal (base 16), the first two digits are the amount of
+red, the second two blue, the third two green, and the final two (if they are
+there) are the alpha value. The values range from 00 in base 16 (0 in base
+10) through to FF (255 in base 10). The alpha represents 
+opacity, where 0 is transparent and 100% is opaque.
 
 #### HSB or HSV colors
-This color space is not quite as well known as RGB, the first number represents
-the hue, the second the saturation (how intense the color is), and the third
+This color space is not quite as well known as RGB. The first number represents
+the hue, the second number the saturation (how intense the color is), and the third
 number the brightness. The image at the top was created by manipulating in this
```

Comment:
Oops. I need to turn this into a figref. I removed many images during the first editing phase, and missed this reference when doing that.